### ISSUE TITLE
[Stupid question] improve readme

### ISSUE THREAD

**Issue by username1222** [xing/kubernetes-oom-event-generator#6]
Hi,

I feel like this is a stupid question; How can I create Prometheus rules from this, and make this useful for my environment?

Can you add something like to the README in an ELI5 fashion?

Viele danke.

**Comment by jabbrwcky** [comment]
Hi,

you could use an alert definition like this one to alert on OOM killed pods:

```
alert: ComponentOutOfMemory
expr: sum_over_time(kube_pod_container_status_terminated_reason{reason="OOMKilled"}[5m])
  > 0
for: 10s
labels:
  severity: warning
annotations:
  description: Critical Pod {{$labels.namespace}}/{{$labels.pod}} was OOMKilled.
```

**Comment by boosty** [comment]
For completeness: `kube_pod_container_status_terminated_reason` is provided by https://github.com/kubernetes/kube-state-metrics.

**Comment by boosty** [comment]
Please have a look at https://github.com/kubernetes/kube-state-metrics/pull/535. `kube_pod_container_status_last_terminated_reason` (notice the `last` part) seems like the better metric to use.
